Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -108,51 +108,61 @@ def signoff!(path, pr: nil, dry_run: false)
     end
   end
 
-  def determine_bump_subject(old_contents, new_contents, formula_path, reason: nil)
-    formula_path = Pathname(formula_path)
-    formula_name = formula_path.basename.to_s.chomp(".rb")
+  def get_package(tap, subject_name, subject_path, content)
+    if subject_path.dirname == tap.cask_dir
+      return begin
+        Cask::CaskLoader.load(content.dup)
+      rescue Cask::CaskUnavailableError
+        nil
+      end
```

Comment:
```suggestion
      cask = begin
        Cask::CaskLoader.load(content.dup)
      rescue Cask::CaskUnavailableError
        nil
      end
      return cask
```

just because `return begin` looks weird.